its/projects/HtmlCSharpExplicitlyMarkedAsTest/HtmlCSharpExplicitlyMarkedAsTest.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Is there any specific reason to have this project on netcore2.2 and not 3.1 as the other one?